app/src/main/java/ch/epfl/sweng/eventmanager/ui/eventShowcase/models/EventShowcaseModel.java

```diff
@@ -42,6 +41,17 @@ public void init(int eventId) {
         return event;
     }
 
+    public LiveData<EventLocation> getLocation() {
+        return Transformations.map(getEvent(), ev -> {
+            if (ev == null || ev.getLocation() == null) {
+                // By default, return EPFL
+                return new EventLocation("EPFL", 46.518510, 6.563249);
```

Comment:
Is it really a good idea to return a random location in case of Null even if the event has nothing to do with EPFL ?

Reply:
Nope